addons/test_mail/tests/test_mail_followers.py

```diff
@@ -46,6 +46,16 @@ def test_field_message_is_follower(self):
         self.assertTrue(test_record.message_is_follower)
         self.assertEqual(followed_before | test_record, followed_after)
 
+    def test_field_message_partner_ids(self):
+        test_record = self.test_record.with_user(self.user_employee)
+        partner = self.user_employee.partner_id
+        followed_before = test_record.search([('message_partner_ids', 'in', partner.ids)])
```

Comment:
'test_field_message_is_follower' is a bit strangely coded actually :D would say 
```suggestion
        followed_before = self.env['mail.test.simple'].search([('message_partner_ids', 'in', partner.ids)])
```
It does the same, but is more clear when reading it, as search is a model method, a bit strange to use a recordset to call search.

Reply:
Thanks for the tips :) Changes done

Anyway IMO this is tested more implicitly in record rules cases (e.g.: in the [project](https://github.com/odoo/odoo/blob/61d1fdf9108ab152cde35e713c8e7a2c48fd234a/addons/project/tests/test_access_rights.py#L79) module)

Reply:
Indeed, but if you change ACLs you lose the coverage, as it is more a 'hidden' test ... here it is clear that we cover the field itself. I think that if the field was broken a lot of tests would be broken, but implicitly broken, not explicitly like this one. And if we work on some code in mail, we generally run 'test_mail' locally before sending to runbot, so it allows to check before testing sub-addons.

Reply:
Good reasoning. Thanks :)